subcommand/sync-catalog/command.go

```diff
@@ -157,21 +161,23 @@ func (c *Command) Run(args []string) int {
 			SyncPeriod:        syncInterval,
 			ServicePollPeriod: syncInterval * 2,
 			ConsulK8STag:      c.flagConsulK8STag,
+			K8sConsulPrefix:   c.flagK8sConsulPrefix,
```

Comment:
If we're able to remove the above `syncer` struct field, this also wouldn't be necessary.